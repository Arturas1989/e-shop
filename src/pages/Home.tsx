export const Home = () => {
  return (
    <main data-testid='home'>Home</main>
  )
};
