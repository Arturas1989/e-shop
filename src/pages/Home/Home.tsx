import { Hero, FeaturedProducts, Testimonials, Faq } from "./index";

export const Home = () => {
  return (
    <main data-testid='home' className="home">
      <Hero />
      <FeaturedProducts />
      <Testimonials />
      <Faq />
    </main>
  )
};
