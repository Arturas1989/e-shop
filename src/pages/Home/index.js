export { Hero } from './components/Hero/Hero';
export { FeaturedProducts } from './components/FeaturedProducts/FeaturedProducts';
export { Testimonials } from './components/Testimonials/Testimonials';