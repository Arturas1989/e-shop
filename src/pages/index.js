export { Home } from './Home/Home';
export { ProductsPage } from './Products/ProductsPage';
export { PageNotFound } from './PageNotFound/PageNotFound';

export { Register } from './Auth/Register/Register';
export { Login } from './Auth/Login/Login';
