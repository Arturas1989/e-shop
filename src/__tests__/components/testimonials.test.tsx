import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Testimonials } from '../../pages/Home';

describe('Testimonial test', () => {
  test('render test', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Testimonials />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});