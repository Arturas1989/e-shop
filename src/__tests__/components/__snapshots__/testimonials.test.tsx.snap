// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonial test render test 1`] = `
<section
  className="testimonials"
>
  <h2
    className="heading"
  >
    Testimonials
  </h2>
  <div
    className="testimonial-cards"
  >
    <div
      className="testimonial-card"
    >
      <div
        className="content-container"
      >
        <h3>
          Excelent
        </h3>
        <p
          className="description"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit, sapiente laboriosam? Eligendi aliquam deleniti, inventore laudantium illo rem in officia.
        </p>
        <div
          className="user"
        >
          <i
            className="bi bi-person-circle"
          />
          <p
            className="name"
          >
            Tom
          </p>
        </div>
      </div>
    </div>
    <div
      className="testimonial-card"
    >
      <div
        className="content-container"
      >
        <h3>
          Nice
        </h3>
        <p
          className="description"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit, sapiente laboriosam? Eligendi aliquam deleniti, inventore laudantium illo rem in officia.
        </p>
        <div
          className="user"
        >
          <i
            className="bi bi-person-circle"
          />
          <p
            className="name"
          >
            John
          </p>
        </div>
      </div>
    </div>
    <div
      className="testimonial-card"
    >
      <div
        className="content-container"
      >
        <h3>
          Best quality
        </h3>
        <p
          className="description"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit, sapiente laboriosam? Eligendi aliquam deleniti, inventore laudantium illo rem in officia.
        </p>
        <div
          className="user"
        >
          <i
            className="bi bi-person-circle"
          />
          <p
            className="name"
          >
            Ben
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;
